creedengojavascript.properties

```diff
@@ -1,12 +1,20 @@
 category=External Analysers
 description=Provide a list of static code analyzers (for Javascript language) to highlight code structures that may have a negative ecological impact: energy and resources over-consumption, "fatware", shortening terminals' lifespan, etc.
 homepageUrl=https://github.com/green-code-initiative/creedengo-javascript
-publicVersions=2.0.0
-archivedVersions=
+publicVersions=2.1.0
+archivedVersions=2.0.0
 
 defaults.mavenGroupId=org.green-code-initiative
 defaults.mavenArtifactId=creedengo-javascript-plugin
 
+2.1.0.description=3 new rules added; Support for SonarQube up to 25.3
+2.1.0.sqs=[10.8,LATEST]
+2.1.0.sqcb=[24.12,LATEST]
+2.1.0.sqVersions=[9.9,10.7.*]
+2.1.0.date=2025-03-31
+2.1.0.downloadUrl=https://github.com/green-code-initiative/creedengo-javascript/releases/download/2.1.0/creedengo-javascript-plugin-2.1.0.jar
+2.1.0.changelogUrl=https://github.com/green-code-initiative/creedengo-javascript/releases/tag/2.1.0
+
 2.0.0.description=Initial Marketplace publication of plugin after renaming from "ecocode-javascript"
 2.0.0.sqs=[10.8,LATEST]
```

Comment:
Archived versions can't be compatible with `LATEST`, and two versions can't list compatibility with `LATEST`. When you archive a public version, change any occurrence of `LATEST` in its compatibility ranges with the current relevant versions. So for SQS, that's 2025.2, and for sqcb that's 25.3.

Reply:
Thank you for the feedback! I have made changes (sorry I was in vacation) 👍